stdlib/public/core/CMakeLists.txt

```diff
@@ -403,18 +403,14 @@ if(BOOTSTRAPPING_MODE STREQUAL "BOOTSTRAPPING")
                     BOOTSTRAPPING 1)
 endif()
 
-set(tooling_stdlib_deps)
-if(TARGET libSwiftScan)
-  list(append tooling_stdlib_deps libSwiftScan)
-endif()
-
 add_swift_target_library(swiftCore
                   ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES}
                   ${swiftCore_common_options}
                   ${compile_flags_for_final_build}
                   FILE_DEPENDS
                     ${swiftCore_common_dependencies}
-                  DEPENDS ${tooling_stdlib_deps}
+                  DEPENDS
+                    libSwiftScan
```

Comment:
The freestanding configurations will likely fail with this since they don't build the tools.

Reply:
(alternatively said, they should not build tools nor tool libraries. They should only build the runtimes and use a pre-built toolchain.)